src/imu/imu.h

```diff
@@ -0,0 +1,62 @@
+#ifndef _IMU_H_
```

Comment:
```suggestion
#ifndef _IMU_H_
```
identifiers that start with a underscore and followed by a capital letter are not valid in C++, see https://stackoverflow.com/questions/19971886/can-an-underscore-be-used-anywhere-in-an-identifier-in-c . However, I see that this is a problem in all icub-tests, so I guess we can fix this in the future.